content/collections/web_experiment/en/targeting.md

```diff
@@ -39,23 +39,43 @@ By default, a new Web Experiment targets all users. Audience targeting enables y
 
 If any segments match, Amplitude buckets that user into a variant based on the configured rollout and variant distribution. For a segment to match, it must meet all conditions you set.
 
-| Parameter | Description |
-| --- | --- |
-| New Users | A new user is first seen after a specified date. When the Web Experiment SDK first sees a user, it stores the date in local storage when the SDK loads then is never updated. May not function as intended shortly after the initial script installation, when all users are new. This **isn't** the same as new user queries in charts and cohorts. |
-| Returning Users | A returning user is a user first seen before a certain date. Same local considerations apply as with New Users. |
-| Referring URL | Matches users who land on your site from a specific referrer. For more information, see [document.referrer](https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer) on MDN. |
-| Landing Page URL | The landing page URL is set once in session storage when the SDK loads. |
-| URL Query Parameters | The current query parameters on the page at the time of evaluation. Commonly used for UTM parameter targeting. |
-| Device Category | Target users by their device type. `Desktop`, `Mobile`, or `Tablet`. |
-| User Agent | Target based on the user agent. Useful for targeting bots. For example, exclude all users where user agent contains `Googlebot`. |
-| Cookies | The cookies in the window at the time of evaluation. |
-| Language | The language set in the user's browser. |
-| Browser | The user's browser: `Safari`, `Chrome`, `Firefox`, `Edge`, `Opera`. |
+### Local properties
+
+Local properties are available on the client-side without the need for network requests and can be evaluated with little to no latency.
```

Comment:
```suggestion
Local properties are available client-side and don't require network requests. This enables Amplitude to evaluate them with low latency.
```

"On the client-side" sounds like it combines "client-side" and "on the client". Went with the first option.